test-app/src/main/java/net/rubygrapefruit/platform/test/Main.java

```diff
@@ -366,25 +368,38 @@ private static void fileSystems() {
         System.out.println("* File systems: ");
         for (FileSystemInfo fileSystem : fileSystems.getFileSystems()) {
             System.out.println(String.format("    * %s -> %s (type: %s %s, case sensitive: %s, case preserving: %s)",
-                    fileSystem.getMountPoint(), fileSystem.getDeviceName(), fileSystem.getFileSystemType(),
-                    fileSystem.isRemote() ? "remote" : "local", fileSystem.isCaseSensitive(), fileSystem.isCasePreserving()));
+                fileSystem.getMountPoint(), fileSystem.getDeviceName(), fileSystem.getFileSystemType(),
+                fileSystem.isRemote() ? "remote" : "local", fileSystem.isCaseSensitive(), fileSystem.isCasePreserving()));
```

Comment:
I noticed these but I'd rather keep them. They are not very disruptive to the review, and if I revert now, we'll reformat again later.